boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_I2C=y
```

Comment:
why is i2c needed?

Reply:
> if not needed, get rid of

> why is i2c needed?

Some peripherals are connected via i2c interface.

Reply:
Once, remove this line.
And let's restore it when drivers are introduced.

Reply:
> Once, remove this line. And let's restore it when drivers are introduced.

Okay, removed.